DependencyInjection/DoctrineExtension.php

```diff
@@ -602,6 +603,13 @@ protected function ormLoad(array $config, ContainerBuilder $container)
             ->addTag('container.preload', [
                 'class' => Autoloader::class,
             ]);
+
+        $container
```

Comment:
why is this registered here and not in orm.xml?

Reply:
yeah can be moved to xml indeed :+1: